src/scripts/templates/index.js

```diff
@@ -3,5 +3,6 @@
 import '../../styles/templates/index.css'
 
 import {load} from '@shopify/theme-sections';
+import '../sections/product';
```

Comment:
Use product.js on the index template? I thought the idea was to use it only on the product page.